.gitignore

```diff
@@ -1 +1,5 @@
-node_modules
\ No newline at end of file
+node_modules
+yarn-error.log
+lerna-debug.log
+yarn.lock
+__tests__/
```

Comment:
Can we remove this rule? This is the folder where we are going to place our unit tests, so I think that we should keep it into the repository.